pages/ButtonColours.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import tableStyles from '../css/tables.css';
+import linkStyles from '../css/link.css';
+import ButtonDiagram from './assets/button-diagram.svg';
+import buttonStyles from '../css/button.css';
+import styles from './css/ButtonPage.css';
+
+function ButtonExample({ label, colour, cssClass, blurb }) {
+  return (
+    <div className={styles['button-section']}>
+      <h4>{label}</h4>
+      <div>{blurb}</div>
+      <br />
+      <div>
+        CSS class: &nbsp;<code>{cssClass}</code>
```

Comment:
```suggestion
        CSS class: <code>{cssClass}</code>
```

Don't need the `&nbsp;` here as there is already a space